flixel/system/frontEnds/BitmapFrontEnd.hx

```diff
@@ -319,7 +319,25 @@ class BitmapFrontEnd
 	{
 		if (_cache == null)
 			_cache = new Map();
-
+		
+		for (key in _cache.keys())
+		{
+			var obj = _cache.get(key);
+			if (obj != null && !obj.persist && obj.useCount <= 0)
```

Comment:
Considering this now checks for `useCount`, the docs probably need to be updated.